docs/src/layouts/SideNavLayout/NavListFilter.scss

```diff
@@ -19,6 +19,9 @@
 
 .bpkdocs-nav-list-filter {
   &__option {
+    margin-top: $bpk-spacing-xs / 3;
+    margin-bottom: $bpk-spacing-xs / 3;
+
```

Comment:
```suggestion
    margin-bottom: $bpk-spacing-sm / 3;

    &:last {
      margin-bottom: 0;
    }
```

Reply:
Maybe the `&` is the thing missing here as I tried updating to do that but everything in the current `:last-child` was having no affect

Reply:
I also added the top margin so that it made it a lot more balanced as just doing the bottom margin made it off balance 😄

Reply:
Turns out it was the missing `&` that caused it so fixed it to actually work now

Reply:
👍